demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb

```diff
@@ -4,14 +4,36 @@
    "cell_type": "markdown",
```

Comment:
<p>I think calling <code>build</code>  is important for how StellarGraph models are used. I suggest a brief explanation of what this call achieves might be useful.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>If that was there before, I must not have seen it. It is ok then.</p>

Reply:
It was there before, so I'll leave it as.